blog/_posts/2021-08-16-write-more-bash.md

```diff
@@ -0,0 +1,206 @@
+---
+slug: "write-more-bash"
+date: 2021-08-16
+author: nick
+layout: blog
+title: "Write More Bash to Hack Features Faster and with Less Testing"
+subtitle: "A walkthrough of the 'cancel' button extension"
+description: "A walkthrough of the 'cancel' button extension"
+image: "/assets/images/write-more-bash/bonk.jpg"
+image_type: "contain"
+tags:
+  - api
+  - bash
+---
+
+The KubeCon schedule was announced last week. The Tilt team is giving two talks
+😇.
+
+- ["Beyond Kubernetes Security"](https://sched.co/lV4f) - A "blockbuster action thriller."
+- ["The Control Loop as an Application Development Framework"](https://sched.co/lV1E) - How to write apps with more control loops.
+
+Both talks are tangentially related to Tilt at best! They're mostly about
+Kubernetes, the ideas behind Kubernetes, and how those ideas can (and should!)
+affect your life as a dev.
+
+In the Control Loop talk, I'm going to talk more about the Kubernetes ideas and
+libraries that Tilt is built on top of. The Kubernetes libraries have a lot of
+tools to help us build robust systems.  But even better, they also have a lot of
```

Comment:
Assuming the "but even better" is a joke, I missed it on the first read. If you wanna get less subtle (which is obviously always a balance with humor), you could do something like this:
```suggestion
tools to help us build robust systems, which, sure, some people find value in.  But far, far more importantly, they also have a lot of
```

Reply:
:stuck_out_tongue: